p2p/simulator/main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"crypto/rand"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	golog "github.com/ipfs/go-log"
+	gologging "github.com/whyrusleeping/go-logging"
+	log "github.com/inconshreveable/log15"
+	p2p "github.com/ChainSafe/gossamer/p2p"
+	peer "github.com/libp2p/go-libp2p-peer"
+)
+
+var messages = []string{
+	"hello friend!\n",
+	"i am a node\n",
+	"do you want to be friends?\n",
+	"ok\n",
+	"noot\n",
+	"i am a penguin stuck in a computer\n",
+	"pls feed me code\n",
+}
+ 
+func sendRandomMessage(s *p2p.Service, peerid peer.ID) error {
+	// open new stream with each peer
+	ps, err := s.DHT().FindPeer(s.Ctx(), peerid)
+	if err != nil {
+		return fmt.Errorf("could not find peer: %s", err)
+	}
+
+	r := getRandomInt(len(messages))
+	msg := messages[r]
+
+	err = s.Send(ps, []byte(msg))
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func getRandomInt(m int) int {
+	b := make([]byte, 1)
+	_, err := rand.Read(b)	
+	if err != nil {
+		return 0
+	}
+	r := int(b[0]) % m
+	return r
+}
+
+func main() {
+	golog.SetAllLoggers(gologging.INFO) // Change to DEBUG for extra info
```

Comment:
Could you please share the documentation you got this from? Seems like unnecessary dependencies

Reply:
good question.... I found it a long time ago. I'll try to dig it up but no guarantees

Reply:
@noot

Reply:
I can't find where I found it, it's in the libp2p issues deep in there

Reply:
@ansermino what do you think, should i just remove it?

Reply:
Yeah, please remove

Reply:
done